app/component/itinerary/navigator/hooks/useRealtimeLegs.js

```diff
@@ -123,24 +123,12 @@ function getLegsOfInterest(
     return acc;
   }, []);
 
```

Comment:
This is now totally unnecessary. Fake wait legs just add complexity. Use realTimeLegs array.